common_module.py

```diff
@@ -335,3 +340,40 @@ def get_jobs_info(fn, write_method=options.write_method):
         }
     )
     return job_ids, usernames, job_states, queue_names
+
+
+def anonymize_func():
+    """
+    creates and returns an _anonymize_func object (closure)
```

Comment:
I think it would be great to add in the docstrings, here and in general, the WHY of things! 
ie. Why do we have anonymization?!